ui-tests/src/main/java/org/openmrs/reference/page/FindPatientPage.java

```diff
@@ -13,7 +13,7 @@
 
     public static final By PATIENT_SEARCH_RESULT = By.id("patient-search-results-table");
     public static final By PATIENT_NAME_SEARCH_RESULT = By.xpath("//table[@id='patient-search-results-table']/tbody/tr/td[2]");
-    public static final By PATIENT_ID_SEARCH_RESULT = By.xpath("//table[@id='patient-search-results-table']/tbody/tr/td[1]");
+    public static final By PATIENT_ID_SEARCH_RESULT = By.cssSelector("#patient-search-results-table td");
```

Comment:
Doesn't it select more than one td? How about "#patient-search-results-table tr:first-child td:first-child" See http://www.w3schools.com/cssref/css_selectors.asp ?